ext/standaloneusers/Civi/Api4/Action/User/WriteTrait.php

```diff
@@ -0,0 +1,107 @@
+<?php
+namespace Civi\Api4\Action\User;
+
+use Civi\API\Exception\UnauthorizedException;
+
+trait WriteTrait {
+
+  /**
+   * If a password is given in the values to write, should we hash it?
+   *
+   * - 'system' when the password you are providing is in plaintext and
+   *    therefore needs to be hashed before it is written to the database
+   * - 'raw' when you want to write the value in password to the database
+   *    as-is. e.g. if you have somehow done your own hashing, or more likely
+   *    you loaded the record from the database and so already have a hashed
+   *    version.
+   * - 'guess' (default) **Do not use this!** It is there to prevent
+   *    hashing-what-was-already-hashed accidents, but you should be explicit
+   *    about what you want the API to do with the password you are providing.
+   *
+   * @var string
+   * @options guess,raw,system
+   * @default guess
+   */
+  protected $passwordHashMode = 'guess';
+
+  /**
+   * If given, this is checked against the current user's password before the change is allowed.
+   *
+   * @var string
+   */
+  protected $actorPassword;
+
+  protected function validateValues() {
+    if ($this->getCheckPermissions()) {
+      // We must have a logged in user...
+      $loggedInUserID = \CRM_Utils_System::getLoggedInUfID() ?? FALSE;
```

Comment:
```suggestion
      $loggedInUserID = \CRM_Utils_System::getLoggedInUfID() ;
```

Redundant coalesce, it just gets cast to boolean next line down.